tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1954,6 +1958,34 @@ def assignDerivedParameters(state):
       state["SynchronizerSizeCheck"] = 1
     #   state["BatchSizeEqual"] = 1
 
+    isa = tuple(state["ISA"])
+    
+    if state["StreamK"] != 0:
+      if state["MIWaveGroup"][0] * state["MIWaveGroup"][1] != 4:
+        reject(state, "Stream-K requries MIWaveGroup0*MIWaveGroup1=4")
+      if state["EnableMatrixInstruction"] and globalParameters["AsmCaps"][isa]["HasWMMA"]:
```

Comment:
I think it is better to reject EnableMatrixInstruction = False

Reply:
Just posted a change to add that reject case, but I also kept one to check for WMMA since that is currently untested